source/client.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react'
 import { render } from 'react-dom'
+import Helmet from 'react-helmet'
 import images from './images'
+import { links, metatags } from './lib/header'
```

Comment:
I maybe wouldn't do this in client...

You could do it in your main layout component, and then it'll run both server (during build) and client-side. But at the moment I don't think this'll be part of the server build ... which is not what you want.